projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/abilities/harvest/autosmelt.json

```
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:autosmelt",
+  "text": [
+    {
+      "text": "业火是绝佳的烧炼热源。"
+    }
+  ],
+  "effects": [
+    "方块的所有掉落物若可被烧炼则直接被烧炼",
```

```suggestion
    "所有方块的掉落物若可被烧炼则直接被烧炼",
```
注意原文，“Drops from all blocks”而不是“All drops from blocks”，我认为这里应当强调的是所有方块都可能被熔炼。